hack/templates/aws_v1alpha1_ccs_accountclaim_cr.tmpl

```diff
@@ -18,12 +18,12 @@ objects:
       regions:
       - name: us-east-1
     awsCredentialSecret:
+      name: aws
       namespace: ${NAMESPACE}
-      name: ${NAME}
     byoc: true
     byocAWSAccountID: ${CCS_ACCOUNT_ID}
     byocSecretRef:
-      name: "byoc"
+      name: byoc
```

Comment:
Just curious about this change. Are the quotes causing issues?

Reply:
No, it was just different from the other templates, and other fields in this template, so I changed it in favor of consistency.  Happy to roll back if needed.

Reply:
Nah just curious :) Sounds reasonable to me